src/Sulu/Bundle/WebsiteBundle/DependencyInjection/Configuration.php

```diff
@@ -56,6 +56,13 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('sitemap')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('dump_dir')->defaultValue('%sulu.cache_dir%/sitemaps')->end()
+                    ->scalarNode('default_host')->defaultNull()->end()
```

Comment:
Shouldn't the default host be defined in a more general location? Might also be useful for other services than the sitemap.